src/main/java/com/facebook/openwifirrm/ucentral/informationelement/TxPwrInfo.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.openwifirrm.ucentral.informationelement;
+
+import java.util.Objects;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+
+/**
+ * This information element (IE) appears in wifiscan entries. It is called
+ * "Tx Pwr Info" in these entries, and "Transmit Power Envelope" in the 802.11
+ * specification. Refer to the specification for more details. Language in
+ * javadocs is taken from the specification.
+ */
+public class TxPwrInfo {
```

Comment:
Is `TxPwrInfo` used for consistency with the wifi scan json? `txPower` is used in more places in the codebase, so I wonder if we can keep them consistent among the codebase.

Reply:
`txPower` is usually used as a method parameter or local variable or something. This is specifically for the Information Element and only describes constraints (max tx power limits)